code-of-conduct/index.md

```diff
@@ -12,4 +12,54 @@ image:
 share: false
 ---
 
-## The code of conduct for the NIPY community 
\ No newline at end of file
+The NIPY community is a community of practice devoted to the use of the Python language in the analysis of neuroimaging data. The following code of conduct reflects our values as a community and serves as a set of guidelines for our individual behaviors in the context of our participation in this community.
```

Comment:
simplify to 'is a guideline for our behavior as we participate in this community'?